gammapy/data/metadata.py

```diff
@@ -124,3 +131,34 @@ def from_header(cls, header, format="gadf"):
         meta = super(GTIMetaData, cls).from_header(header, format)
 
         return meta
+
+
+class EventListMetaData(MetaData):
+    """Metadata containing information about the EventList.
+
+    Parameters
+    ----------
+    """
+
+    _tag: ClassVar[Literal["EventList"]] = "eventlist"
+    event_class: Optional[str] = None
+    creation: Optional[CreatorMetaData] = None
+    optional: Optional[dict] = None
+
+    @classmethod
+    def from_header(cls, header, format="gadf"):
+        meta = super(EventListMetaData, cls).from_header(header, format)
```

Comment:
I think in python 3 `super` does not need arguments 
`meta = super().from_header(header, format)` should work

Reply:
Yes that is true! 
Thanks @AtreyeeS 

I adapted in the other cases too